servicecontrol/dtc-and-transactions.md

```diff
@@ -0,0 +1,21 @@
+---
+title: ServiceControl, MSDTC and transaction support
+summary: ServiceControl uses idempotent message processing and does not rely on MSDTC for exact-once processing but does use transport transactions is supported.
+tags:
+- ServiceControl
+- MSDTC
+---
+
+ServiceControl does not rely on the Microsoft Distributed Transaction Coordinator (MSDTC) for exact-one processing of its messages. Meaning that ServiceControl can be used in environments that do not rely on MSDTC
+
+## Receiving of messages
+
+It processes its messages idempotently when storing messages in its database. Duplicate messages will not result in any issues.
+
+## Sending of messages
+
+ServiceControl can be used to retry messages but also publishes several events. These messages are send as atomic as possible. We have potentially two transactions, the transport transaction and the persistence (RavenDB) transaction. 
```

Comment:
"These messages are send as atomic as possible" 

what exactly is " atomic as possible"?